.github/workflows/main.yaml

```diff
@@ -0,0 +1,20 @@
+name: Main Ci/CD
+on: [push]
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "🎉 The job was automatically triggered by a ${{ github.event_name }} event."
+      - run: echo "🐧 This job is now running on a ${{ runner.os }} server hosted by GitHub!"
+      - run: echo "🔎 The name of branch is ${{ github.ref }} and repository is ${{ github.repository }}."
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - run: echo "💡 The ${{ github.repository }} repository has been cloned to the runner."
```

Comment:
I think it is not useful, because if actions actions/checkout@v3 is failed, all another command will crashed...
And all workflow will crash too)

Reply:
@MariaMaximova, I don't agree with you that it is not useful: if checkout is failed, this step will not run, so it is ok to echo successfull message.
Moreover, in dm you said that steps will not continue to run, but jobs will, so jobs should be restricted for dependencies. So I think we can leave this message as it is :-)